docs/runbooks/LowReadyVirtOperatorsCount.md

```diff
@@ -2,23 +2,53 @@
 
 ## Meaning
 
-This alert is triggered when some virt operators are running but, not in the `Ready` state in the past 10 minutes. The virt-operator deployment has a default replica of 2 pods.
+This alert fires when one or more `virt-operator` pods are running, but none of these pods have been in a `Ready` state for the last 10 minutes. 
```

Comment:
Not with the object attached, AFAIK. I.e., you can't say "one or more pod is running" or "one or more pods is running". The consensus on the subject seems to be rather, well, non-existent, but sounds like it might be more on the side of plural: https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are

Or did I misunderstand what you intended the sentence to say?

Reply:
I guess either is fine.